MoltenVK/MoltenVK/GPUObjects/MVKDescriptorSet.mm

```diff
@@ -707,10 +707,10 @@ static size_t getInlineBlockPoolSize(const VkDescriptorPoolCreateInfo* pCreateIn
 }
 
 // Although poolDescriptors is derived from MVKConfiguration, it is passed in here to ensure all components of this instance see a SVOT for this value.
-// Alternate might have been to force _hasPooledDescriptors to be set first by changing member declaration order in class declaration.
 MVKDescriptorPool::MVKDescriptorPool(MVKDevice* device, const VkDescriptorPoolCreateInfo* pCreateInfo, bool poolDescriptors) :
```

Comment:
As this comment indicates, if we're forcing `_hasPooledDescriptors` to be set first now, there is a bunch of simplifications we can do here.

However, that's a bit out of scope for this PR, so I'll take care of that in a separate PR. Tracking that now in #1640.